### DISCUSSION TITLE
Export numerical edition number instead of text

### DISCUSSION THREAD

**Discussion by jsdodge** [retorquere/zotero-better-bibtex#2416] [Q&A] [answered]
Hello,

I have a book among my references that is in its 7th edition, and the house style of the publisher requires that this be listed as "7th ed.", but the automatic export reformats it as "seventh ed." Is it possible to change this in the export settings?

**Comment by retorquere** [comment]
I'll need a debug log for that entry to be able to say precisely.

**Comment by jsdodge** [comment]
Thanks—I don’t have time right now to follow up on your two suggestions, but will do that in the next few days.

Steve

On Feb 16, 2023, at 10:46 AM, Emiliano Heyns ***@***.******@***.***>> wrote:



I'll need a debug log for that entry to be able to say precisely.

**Comment by jolle** [comment, marked as answer]
I believe this is a feature that was added in https://github.com/retorquere/zotero-better-bibtex/issues/1446, where numerical editions are converted into ordinals. As discussed in the issue, there exists the issue of the ordinal always being in English. I am writing a non-English document, so this annoyed me as well. A temporary workaround I found is that adding a string suffix (that is not "nd" or "th") after the edition number will keep the number intact (with the suffix). The extra suffix seems to be removed in the actual bibliography, at least for me; however, the `.bib` file now has the suffix as well, which is not optimal.

**Reply by retorquere** [reply]
If either one of you would open an issue for this, I think I can devise a workaround.
